OpenEdXMobile/src/main/java/org/edx/mobile/view/dialog/SpeedDialogFragment.java

```diff
@@ -0,0 +1,96 @@
+package org.edx.mobile.view.dialog;
+
+import android.app.DialogFragment;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ListView;
+import android.widget.TextView;
+
+import com.google.inject.Inject;
+
+import org.edx.mobile.R;
+import org.edx.mobile.core.IEdxEnvironment;
+import org.edx.mobile.view.adapters.SpeedAdapter;
+
+import java.util.List;
+
+import roboguice.fragment.RoboDialogFragment;
+
+public class SpeedDialogFragment extends RoboDialogFragment {
```

Comment:
Can we somehow combine this and CCLanguageDialogFragment?

Reply:
Yes, we can combine both the `DialogFragments` but this take more effort then it's worth.

Reply:
got it.